mksproxy_version6.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 Radoslav Gerganov
```

Comment:
Again since this is a new file that we created (albeit by copying the original developer's file and then making modifications), is it appropriate to include the original developer's copyright statement, as well as the licensing statement below?